zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/datastore/IDataConn.scala

```diff
@@ -19,9 +19,50 @@ trait IDataConn {
   def post(statement: String): Future[WSResponse]
 }
 
-class AsterixConn(url: String, wSClient: WSClient)(implicit ec: ExecutionContext) extends IDataConn with Logging {
+class AsterixAQLConn(url: String, wSClient: WSClient)(implicit ec: ExecutionContext) extends IDataConn with Logging {
```

Comment:
this file is a little big and will grow bigger if we have more connections. 
I think we can have a `AsterixConnection` file and put `AQLConn` and `SQLPPConn` there.